src/prefect/artifacts.py

```diff
@@ -100,11 +108,15 @@ async def create_markdown_artifact(
     """
     Create a markdown artifact.
 
-    Args:
-        - markdown: The markdown to create.
+    Arguments:
+        markdown: The markdown to create.
+        key: A user-provided string identifier.
+          Required for the artifact to show in the Artifacts tab in the UI.
```

Comment:
I would probably call it the Artifacts page, as it will still show in the Artifacts tab of the Flow Runs page.

Reply:
done